brother_printer_fwupd/firmware_uploader.py

```diff
@@ -21,7 +21,11 @@ def upload_fw(
     cat LZ5413_P.djf | nc lp.local 9100
     ```
     """
-    port = port if port else socket.getservbyname(PORT_SERVICE_NAME)
+    if port is not None:
```

Comment:
Argh, somehow I pushed this with the condition reversed: should be `is None` 🤦🏻‍♂️